src/constants/routes.ts

```diff
@@ -74,38 +74,56 @@ const paths = {
   version: "/version",
   waterfall: "/waterfall",
 };
+
+export enum slugs {
+  buildId = "buildId",
+  podId = "podId",
+  distro = "distro",
+  distroId = "distroId",
+  hostId = "hostId",
+  patchId = "patchId",
+  projectIdentifier = "projectIdentifier",
+  tab = "tab",
+  taskId = "taskId",
+  taskName = "taskName",
+  variantName = "variantName",
+  versionId = "versionId",
+  userId = "userId",
+}
+
 export const redirectRoutes = {
   distroSettings: paths.distros,
   projectSettings: paths.projects,
-  userPatches: `${paths.user}/:id`,
-  waterfall: `${paths.waterfall}/:projectIdentifier`,
+  userPatches: `${paths.user}/:${slugs.userId}`,
+  waterfall: `${paths.waterfall}/:${slugs.projectIdentifier}`,
 };
 
 export const routes = {
-  commitQueue: `${paths.commitQueue}/:projectIdentifier`,
+  commitQueue: `${paths.commitQueue}/:${slugs.projectIdentifier}`,
   commits: paths.commits,
-  configurePatch: `${paths.patch}/:id/configure`,
-  container: `${paths.container}/:id`,
-  distroSettings: `${paths.distro}/:distroId/${PageNames.Settings}`,
-  host: `${paths.host}/:id`,
+  configurePatch: `${paths.patch}/:${slugs.patchId}/configure`,
+  container: `${paths.container}/:${slugs.podId}`,
+  distroSettings: `${paths.distro}/:${slugs.distroId}/${PageNames.Settings}`,
+  host: `${paths.host}/:${slugs.hostId}`,
   hosts: paths.hosts,
-  jobLogs: `${paths.jobLogs}/:buildId`,
+  jobLogs: `${paths.jobLogs}/:${slugs.buildId}`,
+  jobLogsGroup: `${paths.jobLogs}/:${slugs.taskId}`,
   login: paths.login,
   myPatches: `${paths.user}/${PageNames.Patches}`,
-  patch: `${paths.patch}/:id`,
+  patch: `${paths.patch}/:${slugs.versionId}`,
```

Comment:
should it be `patchId`? (not sure, just guessing since it seems odd)

Reply:
Woah this page should not exist it looks like we have 2 version pages with 2 urls

Reply:
(i saw you tried deleting it but it didn't work, just curious what you found out this page is for)